src/status_im/multiaccounts/create/core.cljs

```diff
@@ -50,8 +44,9 @@
   [{:keys [db] :as cofx}]
   (let [{:keys [selected-id address key-code]} (:intro-wizard db)
         {:keys [address]} (get-selected-multiaccount cofx)
+        hashed-password (ethereum/sha3 (security/safe-unmask-data key-code))
         callback #(re-frame/dispatch [::store-multiaccount-success key-code])]
-    {::store-multiaccount [selected-id address key-code callback]}))
+    {::store-multiaccount [selected-id address hashed-password callback]}))
```

Comment:
maybe better to unmask it in fx ?

Reply:
fxs are suppose to do as little as possible besides the side effect itself, what would be the advantage?